### PR TITLE
Corrected Snack bar message in Login Activity

### LINKED ISSUE #1364: Incorrect Snack bar message in Login Activity
**Summary:** 

When the user presses the login button without entering the credentials, the message shown by snack bar is incorrect.  

**Steps to reproduce:** 

Go to Login Activity, Try to login with empty credentials.

**Expected behaviour:** 

The snack bar message should be like "Please enter the credentials"

**Observed behaviour:** 

The snack bar message is "Invalid username length"

**Device and Android version:** 

Device: RedMi Note 8 Pro
Android Version: 9

**Screenshots:** 

Screenshot_2020-03-01-21-00-43-310_com mifos mifosxdroid 1

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes #1364 

Please Add Screenshots If there are any UI changes.

**Before:**

Screenshot_2020-03-01-21-00-43-310_com mifos mifosxdroid 1

**After:**
Screenshot_2020-03-02-10-35-42-648_com mifos mifosxdroid

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@tarun0 @rachittt Can you please review it?

File: mifosng-android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -799,6 +799,7 @@
     <string name="document_dialog_fragment_loading_message">Uploading document</string>
     <string name="data_table_row_dialog_loading_message">Adding entry to database</string>
     <string name="savings_account_approval_fragment_loading_message">Approving savings account</string>
+    <string name="please_enter_the_credentials">Please enter the credentials</string>
```
Comment (rachittt) [inline mifosng-android/src/main/res/values/strings.xml]:
```suggestion
    <string name="error_enter_credentials">Please Enter the Credentials</string>
```

Comment (RajaVamsi11) [reply mifosng-android/src/main/res/values/strings.xml]: @rachittt Done. Please take a look.

**Comment by RajaVamsi11** [comment]
@rachittt Done.

File: mifosng-android/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -799,6 +799,7 @@
     <string name="document_dialog_fragment_loading_message">Uploading document</string>
     <string name="data_table_row_dialog_loading_message">Adding entry to database</string>
     <string name="savings_account_approval_fragment_loading_message">Approving savings account</string>
+    <string name="please_enter_the_credentials">Please Enter the Credentials</string>
```
Comment (rachittt) [inline mifosng-android/src/main/res/values/strings.xml]:
```suggestion
    <string name="error_enter_credentials">Please Enter the Credentials</string>
```

Comment (RajaVamsi11) [reply mifosng-android/src/main/res/values/strings.xml]: Yes, you are right. I will update it accordingly.

**Pull request merged**